src/Annotations/AppKernel.php

```diff
@@ -283,4 +283,14 @@ public function has($id) : bool
     {
         return isset($this->app[$id]);
     }
+
+    /**
+     * Use only in integration tests.
+     *
+     * @internal
+     */
+    public function override($id, callable $factory)
```

Comment:
Seems like something that should be done when constructing the App in the CI environment?

Reply:
More like a `test` environment like in journal, because the console and the smoke tests should still use a `SqsWatchableQueue` (whether it is backed by goaws or the real AWS)

Reply:
Since we have the `$app['aws']['stub']` flag, we don't need this `override` method.

Reply:
Right, removed